From 69eda7abd42ea5fc7ead47b175a9e0226dfa9594 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Tue, 25 Dec 2012 15:02:05 +0800
Subject: [PATCH 119/602] ENGR00238210-1 imx6 sabresd: update the overlay
 configures

Remove the layout and screenshot fix in JB4.1, which
can not be reproduced on JB4.2.
Enable large heap size for Launcher2.
Remove voice capable from global config, which will
also remove the Phone.apk from build and install.
Enlarge the Downloader data dir size.
Enable WIFI display and dual band support.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 .../keyguard_screen_tab_unlock_land.xml            | 124 ---------------------
 .../base/core/res/res/values-sw720dp/dimens.xml    |  28 -----
 .../frameworks/base/core/res/res/values/config.xml |  31 ++++++
 .../packages/apps/Launcher2/res/values/config.xml  |   3 +
 4 files changed, 34 insertions(+), 152 deletions(-)
 delete mode 100644 imx6/sabresd/overlay/frameworks/base/core/res/res/layout-sw600dp/keyguard_screen_tab_unlock_land.xml
 delete mode 100644 imx6/sabresd/overlay/frameworks/base/core/res/res/values-sw720dp/dimens.xml
 create mode 100644 imx6/sabresd/overlay/packages/apps/Launcher2/res/values/config.xml

diff --git a/imx6/sabresd/overlay/frameworks/base/core/res/res/layout-sw600dp/keyguard_screen_tab_unlock_land.xml b/imx6/sabresd/overlay/frameworks/base/core/res/res/layout-sw600dp/keyguard_screen_tab_unlock_land.xml
deleted file mode 100644
index 8058ee8..0000000
--- a/imx6/sabresd/overlay/frameworks/base/core/res/res/layout-sw600dp/keyguard_screen_tab_unlock_land.xml
+++ /dev/null
@@ -1,124 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2009, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License")
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
-
-<!-- This is the general lock screen which shows information about the
-  state of the device, as well as instructions on how to get past it
-  depending on the state of the device.-->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tabunlock="http://schemas.android.com/apk/res/com.android.tabunlock"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="horizontal"
-    android:id="@+id/root">
-
-    <!-- left side: status and music -->
-    <RelativeLayout
-        android:layout_height="match_parent"
-        android:layout_weight="0.42"
-        android:layout_width="0dip"
-        android:gravity="center">
-
-        <RelativeLayout android:id="@+id/transport_bg_protect"
-            android:layout_width="512dip"
-            android:layout_height="wrap_content">
-
-            <!-- Status -->
-            <include layout="@layout/keyguard_screen_status_land"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginLeft="20dip"
-                android:layout_marginTop="50dip"
-                android:layout_marginRight="30dip"
-                android:layout_alignParentTop="true"
-                android:layout_alignParentLeft="true"/>
-
-            <!-- Music transport control -->
-            <include android:id="@+id/transport"
-                layout="@layout/keyguard_transport_control"
-                android:layout_row="0"
-                android:layout_column="0"
-                android:layout_rowSpan="3"
-                android:layout_columnSpan="1"
-                android:layout_gravity="fill"
-                android:layout_width="match_parent"
-                android:layout_height="512dip"
-                />
-
-        </RelativeLayout>
-
-    </RelativeLayout>
-
-    <!-- right side -->
-    <RelativeLayout
-        android:layout_height="match_parent"
-        android:layout_weight="0.58"
-        android:layout_width="0dip"
-        android:gravity="center_horizontal|center_vertical">
-
-        <TextView
-            android:id="@+id/screenLocked"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:textAppearance="?android:attr/textAppearanceMedium"
-            android:gravity="center"
-            android:layout_marginTop="12dip"
-            android:layout_alignParentLeft="true"
-            android:layout_alignParentTop="true"
-            android:drawablePadding="4dip"/>
-
-        <com.android.internal.widget.multiwaveview.GlowPadView
-            android:id="@+id/unlock_widget"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_rowSpan="7"
-            android:layout_gravity="center_vertical|right"
-            android:gravity="center"
-            android:focusable="true"
-
-            android:targetDrawables="@array/lockscreen_targets_with_camera"
-            android:targetDescriptions="@array/lockscreen_target_descriptions_with_camera"
-            android:directionDescriptions="@array/lockscreen_direction_descriptions"
-            android:handleDrawable="@drawable/ic_lockscreen_handle"
-            android:outerRingDrawable="@drawable/ic_lockscreen_outerring"
-            android:outerRadius="@dimen/glowpadview_target_placement_radius"
-            android:innerRadius="@dimen/glowpadview_inner_radius"
-            android:snapMargin="@dimen/glowpadview_snap_margin"
-            android:feedbackCount="1"
-            android:vibrationDuration="20"
-            android:glowRadius="@dimen/glowpadview_glow_radius"
-            android:pointDrawable="@drawable/ic_lockscreen_glowdot"
-        />
-
-        <!-- emergency call button shown when sim is PUKd and tab_selector is hidden -->
-        <Button
-            android:id="@+id/emergencyCallButton"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginRight="80dip"
-            android:layout_marginBottom="80dip"
-            android:layout_alignParentRight="true"
-            android:layout_alignParentBottom="true"
-            android:drawableLeft="@drawable/ic_emergency"
-            style="@style/Widget.Button.Transparent"
-            android:drawablePadding="8dip"
-            android:visibility="gone"/>
-
-    </RelativeLayout>>
-
-</LinearLayout>
diff --git a/imx6/sabresd/overlay/frameworks/base/core/res/res/values-sw720dp/dimens.xml b/imx6/sabresd/overlay/frameworks/base/core/res/res/values-sw720dp/dimens.xml
deleted file mode 100644
index 2418175..0000000
--- a/imx6/sabresd/overlay/frameworks/base/core/res/res/values-sw720dp/dimens.xml
+++ /dev/null
@@ -1,28 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
-**
-** Copyright 2012, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
--->
-<resources>
-
-    <!-- The width that is used when creating thumbnails of applications. -->
-    <dimen name="thumbnail_width">230dp</dimen>
-    <!-- The height that is used when creating thumbnails of applications. -->
-    <dimen name="thumbnail_height">170dp</dimen>
-
-</resources>
-
-
diff --git a/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml b/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
index 3c32eb1..21fa3f4 100755
--- a/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
@@ -157,4 +157,35 @@
          autodetected from the Configuration. -->
     <bool name="config_showNavigationBar">true</bool>
 
+    <!-- This device is not "voice capable"; it's data-only. -->
+    <bool name="config_voice_capable">false</bool>
+
+    <!-- Max space (in MB) allocated to DownloadManager to store the downloaded files
+         if they are to be stored in DownloadManager's data dir,
+         which typically is /data/data/com.android.providers.downloads/files. -->
+    <integer name="config_downloadDataDirSize">200</integer>
+
+    <!-- When the free space available in DownloadManager's data dir falls
+         below the percentage value specified by this param, DownloadManager
+         starts removing files to try to make percentage of available
+         free space above this threshold value. -->
+    <integer name="config_downloadDataDirLowSpaceThreshold">10</integer>
+
+    <!-- Boolean indicating whether the wifi chipset has dual frequency band support -->
+    <bool translatable="false" name="config_wifi_dual_band_support">true</bool>
+
+    <!-- Whether WiFi display is supported by this device.
+         There are many prerequisites for this feature to work correctly.
+         Here are a few of them:
+         * The WiFi radio must support WiFi P2P.
+         * The WiFi radio must support concurrent connections to the WiFi display and
+           to an access point.
+         * The Audio Flinger audio_policy.conf file must specify a rule for the "r_submix"
+           remote submix module.  This module is used to record and stream system
+           audio output to the WiFi display encoder in the media server.
+         * The remote submix module "audio.r_submix.default" must be installed on the device.
+         * The device must be provisioned with HDCP keys (for protected content).
+    -->
+    <bool name="config_enableWifiDisplay">true</bool>
+
 </resources>
diff --git a/imx6/sabresd/overlay/packages/apps/Launcher2/res/values/config.xml b/imx6/sabresd/overlay/packages/apps/Launcher2/res/values/config.xml
new file mode 100644
index 0000000..3b82809
--- /dev/null
+++ b/imx6/sabresd/overlay/packages/apps/Launcher2/res/values/config.xml
@@ -0,0 +1,3 @@
+<resources>
+    <bool name="config_largeHeap">true</bool>
+</resources>
-- 
1.8.0

